frontend/src/views/DashboardManager/EgalimProgression.vue

```diff
@@ -199,7 +200,9 @@ export default {
       return null
     },
     approDiagnostic() {
-      return this.centralDiagnostic || this.canteenDiagnostic
+      if (this.isCurrentYear && this.purchasesSummary && this.purchasesSummary.valueTotalHt !== 0) {
+        return this.purchasesSummary
+      } else return this.centralDiagnostic || this.canteenDiagnostic
```

Comment:
```suggestion
      }
      return this.centralDiagnostic || this.canteenDiagnostic
```
pas besoin du else :)